bucket/everything-cli.json

```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://www.voidtools.com/",
+    "description": "Command line interface to Everyting.",
+    "license": "MIT",
```

Comment:
Where did you get this information?

Reply:
From the source code included in the download. It doesn't explicitly say MIT, but it's the MIT license text.